pip install is broken

tables just released a new version 2 hours ago that breaks pip install for allensdk. To reproduce:


```bash
conda create -n test
conda activate test
conda install pip -y
pip install allensdk
```
error:
```
...
Collecting tables (from allensdk)
  Downloading https://files.pythonhosted.org/packages/f9/cb/dfca7ec0f97d34bc9acd6e1658d5c5d3642dff5c453d65ee239eda82dcdd/tables-3.5.0.tar.gz (8.3MB)
    100% |████████████████████████████████| 8.3MB 7.0MB/s 
    Complete output from command python setup.py egg_info:
    /var/folders/mc/k9f_f9bn6h11j5t97_12s2jc0000gn/T/H5close_8cjgr5d.c:1:1: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
    main (int argc, char **argv) {
    ^
    /var/folders/mc/k9f_f9bn6h11j5t97_12s2jc0000gn/T/H5close_8cjgr5d.c:2:5: warning: implicit declaration of function 'H5close' is invalid in C99 [-Wimplicit-function-declaration]
        H5close();
        ^
    2 warnings generated.
    ld: library not found for -lhdf5
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    * Using Python 3.7.2 (default, Dec 29 2018, 00:00:04)
    * USE_PKGCONFIG: False
    * Found conda env: ``/Users/bendichter/anaconda3/envs/test``
    .. ERROR:: Could not find a local HDF5 installation.
       You may need to explicitly state where your local HDF5 headers and
       library can be found by setting the ``HDF5_DIR`` environment
       variable or by using the ``--hdf5`` command-line option.
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/mc/k9f_f9bn6h11j5t97_12s2jc0000gn/T/pip-install-2ckx6lyl/tables/
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -14,4 +14,4 @@ scikit-image>=0.14.0
 scikit-build
 statsmodels>=0.8.0
 simpleitk
-tables
+tables==3.4.4

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,6 @@
 h5py>=2.8
 matplotlib>=1.4.3
-numpy>=1.15.1
+numpy==1.15.4
 pandas>=0.23.0
 jinja2>=2.7.3
 scipy>=0.15.1

